src/index.ts

```diff
@@ -216,13 +215,16 @@ export class EsbuildPlugin implements Plugin {
     });
   }
 
-  /** Link or copy extras such as node_modules or package.include definitions */
+  /** Link or copy extras such as node_modules or package.patterns definitions */
   async copyExtras() {
     const { service } = this.serverless;
 
-    // include any "extras" from the "include" section
-    if (service.package.include && service.package.include.length > 0) {
-      const files = await globby(service.package.include);
+    // include any "extras" from the "patterns" section
+    const globalPatterns = [
+      ...new Set([ ...(service.package.include || []), ...(service.package.patterns || []) ]),
```

Comment:
Just for code clarity, what do you think of putting this at the same place where you merge exclude and patterns? (l.180). Otherwise it's good for me.

Reply:
Yeah, good point, I've updated this.